aliasMultinomial.lua

```diff
@@ -0,0 +1,13 @@
+local AM = torch.class('torch.AliasMultinomial') 
```

Comment:
@soumith Does this seem okay ?```gen = torch.aliasmultinomial_generator(probs)
out = torch.aliasmultinomial(gen, n_sample ) ``` 
As @pavanky  suggested, instead of passing the output tensor, we should pass the size instead